projects/ngx-google-analytics/package.json

```diff
@@ -1,13 +1,17 @@
 {
   "name": "ngx-google-analytics",
-  "version": "13.0.0",
+  "version": "13.0.1",
   "description": "A simple ng-9 wrapper to load Google Analytics dependency by angular way",
   "keywords": [ "google", "google analytics", "angular", "angular-9+", "typescript" ],
   "homepage": "https://github.com/maxandriani/ngx-google-analytics",
   "license": "MIT",
   "dependencies": {
     "tslib": "^2.0.0"
   },
+  "peerDependencies": {
+    "@angular/common": "^13.2.2",
```

Comment:
That's a very restrictive version range. The library should work just fine with Angular v12 and higher.

Reply:
Updated. Thanks for the patience reviewing this
Also finished giving it a test on my side, and everything seems to be working fine